force-app/main/default/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -530,13 +532,21 @@ export default class GeFormRenderer extends LightningElement{
 
     @wire(getRecord, {
         recordId: '$batchId',
-        fields: [FORM_TEMPLATE_FIELD, BATCH_DEFAULTS_FIELD]
+        fields: [FORM_TEMPLATE_FIELD,
+            BATCH_DEFAULTS_FIELD],
+        optionalFields: [DATA_IMPORT_BATCH_ALLOW_RECURRING_DONATIONS]
     })
     wiredBatch({data, error}) {
         if (data) {
-            this.formTemplateId = data.fields[apiNameFor(FORM_TEMPLATE_FIELD)].value;
-            this._batchDefaults = data.fields[apiNameFor(BATCH_DEFAULTS_FIELD)].value;
-            GeFormService.getFormTemplateById(this.formTemplateId)
+            this._batch[apiNameFor(BATCH_DEFAULTS_FIELD)] =
+                data.fields[apiNameFor(BATCH_DEFAULTS_FIELD)
+                    ].value;
+            this._batch[
+                apiNameFor(DATA_IMPORT_BATCH_ALLOW_RECURRING_DONATIONS)] =
+                data?.fields[apiNameFor(DATA_IMPORT_BATCH_ALLOW_RECURRING_DONATIONS)
+                    ]?.value;
```

Comment:
```suggestion
            this._batch[apiNameFor(BATCH_DEFAULTS_FIELD)] =
                data.fields[apiNameFor(BATCH_DEFAULTS_FIELD)].value;
            this._batch[apiNameFor(DATA_IMPORT_BATCH_ALLOW_RECURRING_DONATIONS)] =
                data?.fields[apiNameFor(DATA_IMPORT_BATCH_ALLOW_RECURRING_DONATIONS)]?.value;

```

This wired getRecord is overdue for refactor 😱